### ISSUE TITLE
Greek letters as symbols a11y update

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#4238]
Greek letters are used in articles, sometimes in the text to refer to items (e.g. mathematics or linguistics) and often for units (e.g. micrograms, nanometres).  These need to be marked-up for screenreaders appropriate to their context (e.g. the former might be 'mu' but the later might be 'micrograms').

We need to consider how these are presented in the JATS in order to interpret them the way the author expects and intends.

**Comment by StephDriver** [comment]
from #4194:
- Sample 2, item 13. <i>ɛna</i> which has been put in like that, with the symbol, and so voiceover reads it as 'na' unable to parse the epsilon.
- Sample 2, item 27. The symbols aren't read by voiceover, making, for example, this section unintelligible. This issue continues throughout the article.
- Sample 4, item 29. standardized to 2,000 μg/mL of protein. the mu was not read as micro. Note the mu is read out in a different voice (Greek voice?) and so it is very difficult to recognise. I expected "micrograms per millilitre" It is being read as "mu g slash m L". Slash is generally not being read as 'per'.
- Sample 4, item 31. An α of 0.05 was set for a level of significance. this is read as 'An greek letter alpha of ..." is that the desired behaviour?

**Comment by StephDriver** [comment]
A11y Themed Sprint Triage:

- this is about user content, put to Phase 2.

The unicode solution for Roman Numerals from #4223 does not seem applicable here as the Greek Letters are used for parts of units (e.g. nanometres) and so the item to communicate is the whole unit, not just the nano part.  But the solution is likely to involve typesetting behaviour, so that makes it user-content and Phase 2.
